Reachability/Reachability.swift

```diff
@@ -71,6 +71,8 @@ public class Reachability {
     public var whenReachable: NetworkReachable?
     public var whenUnreachable: NetworkUnreachable?
     public var reachableOnWWAN: Bool
+    
+    public var notificationCenter: NotificationCenter = NotificationCenter.default
```

Comment:
OK, thanks. So future updaters know why this has been done, please add a comment to the property to explain and I'll merge